tools/lsp/preview/properties.rs

```diff
@@ -753,15 +778,15 @@ mod tests {
             "enum AccessibleRole"
         );
         // Accessible property should not be present since the role is none
-        assert_eq!(find_property(&result, "accessible-label"), None);
-        assert_eq!(find_property(&result, "accessible-action-default"), None);
+        assert!(find_property(&result, "accessible-label").is_none());
+        assert!(find_property(&result, "accessible-action-default").is_none());
```

Comment:
(the adventage of assert_eq is that it shows `Some(...)` in the log when the test failes because it isn't None)

Reply:
True, but for that I need to make `CodeBlockOrExpression` support `PartialEq`. That seemed like too much work to keep the `assert_eq` lines.